scripts/ida_toggles.py

```diff
@@ -83,7 +128,7 @@ def add_annotations(self):
         )
         for toggle_type in self.toggles.keys():
             annotation_count = len(
-                list(filter(lambda t: t.annotations, self.toggles[toggle_type]))
+                list(filter(lambda t: t.annotations, [value for key, value in self.toggles[toggle_type].items()]))
```

Comment:
Is this different than: `list(self.toggles[toggle_type].values())`?

Reply:
I simplified the entire block to `([value.annotations for key, value in self.toggles[toggle_type].items() if value.annotations]`

Reply:
Nit: If you aren't using `key`, you can use `_` instead.